core/modules/user/tests/user_password_reset.test

```diff
@@ -204,4 +207,87 @@ class UserPasswordResetTest extends BackdropWebTestCase {
     $this->assertNoText('You have tried to use a reset password link that has either been used or is no longer valid. Please request a new one using the form below.');
   }
 
+  /**
+   * Test user-based flood control on password reset.
+   */
+  public function testPasswordResetFloodControlPerUser() {
+    // Set a very low limit for testing.
+    config_set('user.flood', 'flood_user_limit', 2);
+
+    // Create a user.
+    $account = $this->backdropCreateUser();
+    $this->backdropLogin($account);
+    $this->backdropLogout();
+
+    $edit = array('name' => $account->name);
```

Comment:
Is this needed here? I'm looking at `user_pass()` and the function automatically sets the account name in a hidden field when you are logged in.

Reply:
This *is* necessary. The form field may not be "visible" to users, but the value has to be in the POST (done by backdropPost()).

Reply:
Yes, of course. Not sure what I was thinking :)